src/script/page/LeftSidebar/panels/Conversations/Conversations.test.tsx

```diff
@@ -34,49 +34,35 @@ describe('Conversations', () => {
       conversationLabelRepository: {
         addEventListener: jest.fn(),
         removeEventListener: jest.fn(),
+        getFavorites: jest.fn().mockReturnValue([]),
+        getLabels: jest.fn().mockReturnValue([]),
+        getLabelConversations: jest.fn().mockReturnValue([]),
+        labels: [],
       },
     } as any,
     listViewModel: {
+      switchList: jest.fn(),
       contentViewModel: {
         loadPreviousContent: jest.fn(),
+        switchContent: jest.fn(),
       },
     } as any,
-    preferenceNotificationRepository: {notifications: ko.observable([])} as any,
+    preferenceNotificationRepository: {notifications: observable([])} as any,
     propertiesRepository: {getPreference: jest.fn(), savePreference: jest.fn()} as any,
     selfUser: new User(),
-    switchList: jest.fn(),
+    integrationRepository: {integrations: observable([])} as any,
+    searchRepository: {search: jest.fn()} as any,
+    teamRepository: {getTeam: jest.fn()} as any,
+    userRepository: {users: observable([])} as any,
   };
 
   it('Opens preferences when clicked', () => {
-    const {getByTitle} = render(<Conversations {...defaultParams} />);
-    const openPrefButton = getByTitle('tooltipConversationsPreferences');
+    const {getByTitle} = render(withTheme(<Conversations {...defaultParams} />));
+    const openPrefButton = getByTitle('preferencesHeadline');
     act(() => {
       openPrefButton.click();
     });
 
-    expect(defaultParams.switchList).toHaveBeenCalledWith(ListState.PREFERENCES);
-  });
-
-  it('Switches between folder and list view and save view state', () => {
```

Comment:
This behavior is not relevant anymore, "list view" and "folder view" are not distinct anymore